src/k-provider/ovp/provider.js

```diff
@@ -335,7 +335,7 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
     if (mediaEntry.sources.captions) {
       sourcesObject.captions = mediaEntry.sources.captions;
     }
-    if (mediaEntry.metadata && typeof mediaEntry.metadata.tags === 'string' && mediaEntry.metadata.tags.search(/\b360\b/) > -1) {
+    if (mediaEntry.metadata && typeof mediaEntry.metadata.tags === 'string' && mediaEntry.metadata.tags.split(', ').includes('360')) {
```

Comment:
in case of tag with space for example: "xyz 360 xyz" it identify this tag as 360 and activate the vr.
because we get the all tags as one string the regex returns true as it find string of "360" even if the tag is not the exact match.

Reply:
got it.